README.md

```diff
@@ -36,6 +36,9 @@ Currently supports the following CI servers:
 ## Example config with all possible options
 ```yaml
 title: Example projects
+fetchFrequency: # optional
```

Comment:
This is not what I meant exactly but it's also nice, as it's clear that it's minutes, so I am happy with it